### ISSUE TITLE
The MediaKeySessionConfiguration type definition misses the "sessionTypes" property

### ISSUE THREAD

**Issue by peaBerberian** [microsoft/TypeScript#19189]
**TypeScript Version:**  2.6.0

Hi, As the title suggest, the ``MediaKeySessionConfiguration`` type, which is used in [EncryptedMediaExtensions APIs](https://www.w3.org/TR/encrypted-media/) on supported browsers, seems to miss its ``sessionTypes`` property, which should be an array of string as defined [here](https://www.w3.org/TR/encrypted-media/#mediakeysystemconfiguration-dictionary).

I checked all definition files where it is defined (by the way I find it hard to know on which ``es[x].d.ts`` file it should be defined, since this is more of a W3C spec than a ECMA one), only this property seems to be missing.

**Code**

```ts
function foo(x : MediaKeySystemConfiguration) : string[] {
  return x.sessionTypes;
}
```

**Expected behavior:**
Should work.

**Actual behavior:**
TypeScript complains that:
```Property 'sessionTypes' does not exist on 'MediaKeySystemConfiguration'.```

**Comment by mhegazy** [comment]
PRs welcomed. You can find more information about contributing lib.d.ts fixes at https://github.com/Microsoft/TypeScript/blob/master/CONTRIBUTING.md#contributing-libdts-fixes.

**Comment by aastronaut** [comment]
Seems like that issue has been resolved already with commit `2f9218f346b` by @sandersn

**Comment by jakebailey** [comment]
Just looking at old issues; yes, this was fixed a while ago (as noted above).
